test/Altinn.Notifications.IntegrationTests/Notifications/EmailNotificationsOrderController/EmailNotificationOrdersControllerTests.cs

```diff
@@ -171,6 +171,7 @@ public async Task Post_ServiceReturnsError_ServerError()
 
         // Act
         HttpResponseMessage response = await client.SendAsync(httpRequestMessage);
+        string respoonseString = await response.Content.ReadAsStringAsync();
```

Comment:
Can this line be deleted? If not, typo should be fixed and string value used in some sort of assertion